### DISCUSSION TITLE
i am getting this error installing a kkp cluster any hints

### DISCUSSION THREAD

**Discussion by bassemtadros** [kubermatic/kubermatic#12890] [General]
failed to reconcile Addon "default-storage-class": failed to deploy the addon manifests into the cluster: failed to execute '/usr/local/bin/kubectl-1.27 --kubeconfig /tmp/cluster-tqmhxwh5fq-addon-default-storage-class-kubeconfig apply --prune --filename /tmp/cluster-tqmhxwh5fq-default-storage-class.yaml --selector kubermatic-addon=default-storage-class' for addon default-storage-class of cluster tqmhxwh5fq: exit status 1 storageclass.storage.k8s.io/cinder-csi unchanged Error from server (InternalError): error when creating "/tmp/cluster-tqmhxwh5fq-default-storage-class.yaml": Internal error occurred: failed calling webhook "validation-webhook.snapshot.storage.k8s.io": failed to call webhook: Post "https://snapshot-validation-service.kube-system.svc:443/volumesnapshot?timeout=2s": No agent available

**Comment by embik** [comment]
Closing this as we also discussed it in #12891. For reference, #12304 is already tracking this.
